src/config/dynamic-theme-fixes.config

```diff
@@ -3693,6 +3693,25 @@ article, article section, article footer, .css-1fzo8jw, .css-efdc07, article .cs
 
 ================================
 
+brack.ch
+
+CSS
+body, body > header, .popover, .popover-body {
+    background: var(--darkreader-neutral-background) !important;
+    color: var(--darkreader-neutral-text) !important;
+}
+.list-group-item, .modal-content {
+    background: var(--darkreader-neutral-background) !important;
+}
+a:not(.product__category):not(:hover) {
+    color: var(--darkreader-neutral-text) !important;
+}
+.bhaal1Vl:hover, .bhauyhAt:hover {
```

Comment:
Everything seems to work ok (at least on main page) but I can not find this classes there. Can you share URL (and screenshot if possible) that I could check how page works with and without this additional fix?